src/BootstrapAdminUi/templates/shared/grid/action/delete.html.twig

```diff
@@ -1,5 +1,10 @@
 {% set id = options.link.parameters['id']|default(data.id)  %}
 
-{% set path = options.link.url|default(path(options.link.route|default(grid.requestConfiguration.getRouteName('delete')), options.link.parameters|default({'id': id}))) %}
+{% if options.link.url is defined %}
+    {% set path = options.link.url %}
+{% else %}
+    {% set default_route = options.link.route|default(grid.requestConfiguration is defined ? grid.requestConfiguration.getRouteName('delete') : null) %}
```

Comment:
`grid.requestConfiguration` is defined only when using the `GridViewFactory` from the Resource bundle. It's not working with the `GridViewFactory` from the Grid bundle one.
We'll need that fix for the future Grid as Live component.